ui/src/CaseDetails.js

```diff
@@ -149,6 +149,7 @@ class CaseDetails extends Component {
           {uacQidLink.active ? "Yes" : "No"}
         </TableCell>
         <TableCell component="th" scope="row">
+          {JSON.stringify(uacQidLink.uacMetadata)}
```

Comment:
you have merged this incorrectly. It needs a whole `TableCell` of its own... not to be bundled in with `eqLaunched`